nhrpd/nhrp_peer.c

```diff
@@ -279,11 +280,34 @@ static int nhrp_peer_request_timeout(struct thread *t)
 	return 0;
 }
 
+static int nhrp_peer_defer_vici_request(struct thread *t)
+{
+	struct nhrp_peer *p = THREAD_ARG(t);
+	struct nhrp_vc *vc = p->vc;
+	struct interface *ifp = p->ifp;
+	struct nhrp_interface *nifp = ifp->info;
+	char buf[256];
```

Comment:
Do you need more than SU_ADDRSTRLEN bytes here ? if not then please avoid using hardcoded number and just use SU_ADDRSTRLEN. See examples in other places in the code.